app/models/vm_or_template/operations/relocation.rb

```diff
@@ -60,6 +60,18 @@ def migrate(host, pool = nil, priority = "defaultPriority", state = nil)
     raw_migrate(host, pool, priority, state)
   end
 
+  def migration_status
+    miq_custom_get(:migration_status)
+  end
+
+  def reset_migration_status
+    miq_custom_delete(:migration_status)
+  end
+
+  def set_migration_status(value)
```

Comment:
I don't think we need a `set` method in the base class.

Reply:
As above we can move but check for the method is needed.